src/entries/Dashboard/Dashboard.jsx

```diff
@@ -51,6 +59,7 @@ export default function Dashboard(props) {
   const [isConfigLoaded, setIsConfigLoaded] = useState(false);
   const [windowWidth, setWindowWidth] = useState(window.innerWidth);
   const [isShowPatientDetailsOpen, setPatientDetailsOpen] = useState(false);
+  document.title = "IPD";
```

Comment:
why do we need this change?

Reply:
I added it initially, but removed later on, I think it came back after some rebasing. will remove it.